specutils/spectra/spectral_region.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import astropy.units as u
+
+
+class SpectralRegion:
+    """
+    A `SpectralRegion` is a container class enables some simplicty
+    in defining and passing a region (interval) for a spectrum.
+
+    In the future, there might be more functionality added in here and there
+    is some discussion that this might/could move to
+    `Astropy Regions <http://astropy-regions.readthedocs.io/en/latest/>`_.
+    """
+
+    @staticmethod
+    def from_center(center=None, width=None):
```

Comment:
I would think this should be a *class* method, rather than a static method, is this is basically what classmethod was invented for.  That is:
```
@classmethod
def from_center(cls, center=None, width=None):
    ...
    return cls(center - width, center + width)
```
that way sub-classes still get the correct `from_center` behavior.